components/__tests__/Template-test.js

```diff
@@ -37,7 +37,7 @@ describe('Template', () => {
     renderer.render(<Template {...props} />);
     let out = renderer.getRenderOutput();
 
-    expect(out).toEqualJSX(<div dangerouslySetInnerHTML={{__html: 'it works with strings'}}></div>);
+    expect(out).toEqualJSX(<div className={undefined} dangerouslySetInnerHTML={{__html: 'it works with strings'}}></div>);
```

Comment:
That's just how we implemented it but `className={undefined}` is not the same as no className. Like `o = {a: undefined}` is not the same as `o = {}`